misc/cf-support

```diff
@@ -182,6 +182,41 @@ if [ -f $WORKDIR/share/cf-support-nova-hub.sh ]; then
   source $WORKDIR/share/cf-support-nova-hub.sh
 fi
 
+echo "Analyzing CFEngine core dumps"
+_core_log="$tmpdir"/core-dump.log
+cf_core_files=$(find / -name 'core*' -type f -exec file {} \; 2>/dev/null | grep "core file" | grep "cf-" | cut -d' ' -f1 | sed 's/:$//')
```

Comment:
Would be nice to add a sample line that is being processed in this pipeline.

Reply:
Right, a sample line as a comment. Ack.